dictionaries/dict-en-AU-with-extra-stroke.json

```diff
@@ -85,7 +85,6 @@
 "ABG/KHRAOEUPL/TEU/SA*EUGS/A*U": "acclimatisation",
 "ABG/SES/RAOEUZ/A*U": "accessorise",
 "ABG/TKPWROE/A*U": "aggro",
-"AEFLD/A*U": "realised",
```

Comment:
The correct definition (`RAEFLD/A*U`) already exists in this dict, so just removing the misstroke.